ci/ci.sh

```diff
@@ -135,6 +135,7 @@ test_core() {
 prepare_docker() {
     rm "${WORKSPACE_DIR}"/python/dist/* ||:
     pushd "${WORKSPACE_DIR}/python"
+    pip install -e . --verbose
```

Comment:
is this intentional?

Reply:
Yes. When I'm doing the testing, I'm not sure, but it seems that without this, the image I've built will be wrong. I hope it's ok just add it here. (also ok with removing it and add it in other prs).